src/screenshots/capturer.js

```diff
@@ -101,12 +102,15 @@ export default class Capturer {
         return joinPath(imageDir, 'thumbnails', imageName);
     }
 
-    async _takeScreenshot (filePath, pageWidth, pageHeight) {
+    async _takeScreenshot (filePath, pageWidth, pageHeight, fullPage) {
```

Comment:
I guess this function is private so let's refactor it to a parameter object style.